server/option.go

```diff
@@ -150,7 +157,8 @@ func (o *scheduleOption) RemoveSchedulerCfg(name string) error {
 			return errors.Trace(err)
 		}
 		if tmp.GetName() == name {
-			v.Schedulers = append(v.Schedulers[:i], v.Schedulers[i+1:]...)
+			schedulerCfg.Disable = true
```

Comment:
I think it's okay to just remove instead of setting disable.

Reply:
we need to record the state, otherwise, it will create this scheduler after a restart.  Behave like we add a new default scheduler.